maintainers/maintainer-list.nix

```diff
@@ -14112,4 +14112,10 @@
     github = "nigelgbanks";
     githubId = 487373;
   };
+  cigrainger = {
```

Comment:
Sorry to be so nitpicky, but: despite there being some other entries ignoring this request, please keep the list alphabetically sorted: https://github.com/NixOS/nixpkgs/blob/56a682817d68050298b9fb0c56628b5a85441ad4/maintainers/maintainer-list.nix#L49

Reply:
oh yeah sorry I missed that! we should prob add a lint for this at some point

Reply:
I'll make the change. 👍

Reply:
All done, sorry about that!